Lib/test/test_sysconfig.py

```diff
@@ -612,6 +611,8 @@ def test_always_set_py_gil_disabled(self):
         self.assertEqual(Py_GIL_DISABLED, support.Py_GIL_DISABLED)
 
     def test_abiflags(self):
+        # XXX: If this test fails on some platforms, maintainers should add/update
```

Comment:
```suggestion
        # If this test fails on some platforms, maintainers should add/update
```

This isn't somewhere we'd use an "XXX" (or HACK/TODO) marker.